idseq_dag/steps/run_lzw.py

```diff
@@ -44,6 +44,7 @@ def lzw_fraction(sequence):
         return float(len(results)) / len(sequence)
 
     @staticmethod
+    @command.run_in_subprocess
```

Comment:
This is unlikely to be helpful, unfortunately.   I am preparing a comment with a suggestion.

Reply:
Here is my suggestion:   https://gist.github.com/boris-dimitrov/ec6675dc72957d8991c2bd8dc427067a